File: 2330-valid-palindrome-iv/2330-valid-palindrome-iv.py
class Solution:
    def makePalindrome(self, s: str) -> bool:
        low, high, count = 0, len(s)-1, 0;
        while(low < high):
            if(s[low] != s[high]):
                count+=1
            if(count > 2):
                return False
            low, high = low+1, high-1
        return True

File: 2330-valid-palindrome-iv/2330-valid-palindrome-iv.java
class Solution {
    public boolean makePalindrome(String s) {
        int low = 0, high = s.length()-1, count = 0;
        while(low < high){
            if(s.charAt(low) != s.charAt(high))
                count++;
            if(count > 2)
                return false;
            low += 1; high -= 1;
        }
        return true;
    }
}